.github/workflows/coverage-test.yml

```diff
@@ -0,0 +1,43 @@
+name: Coverage-test
+
+# Controls when the action will run. 
```

Comment:
These comments are explaining how GitHub workflow works. Since we are not using this as a tutorial, can you remove these comments?